requirements/base.in

```diff
@@ -1,7 +1,11 @@
 # Core requirements for using this application
 -c constraints.txt
 
-Django             # Web application framework
+Django                      # Web application framework
+djangorestframework         # Django REST framework for API development
 
 
 openedx-atlas
+casbin-django-orm-adapter   # Casbin Django ORM adapter
+django_authorization        # Django Authorization library
+redis-watcher               # Redis Watcher for Casbin
```

Comment:
I'd like the dependency graph of this so we understand what each library is used for.